src/main/java/com/coreos/jetcd/exception/ClosedSnapshotException.java

```diff
@@ -0,0 +1,19 @@
+package com.coreos.jetcd.exception;
+
+public class ClosedSnapshotException extends EtcdException {
+
+  private static final ClosedSnapshotException DEFAULT_INSTANCE;
+
+  static {
+    DEFAULT_INSTANCE = new ClosedSnapshotException();
```

Comment:
I guess I will just have to create a new instance of the `ClosedSnapshotException` every time then.